libindi/drivers/telescope/ieqprodriver.cpp

```diff
@@ -910,8 +910,8 @@ bool set_ieqpro_guide_rate(int fd, double rate)
     int nbytes_read    = 0;
     int nbytes_written = 0;
 
-    int num = rate * 1000;
-    snprintf(cmd, 16, ":RG%03d#", num);
+    int num = rate * 100;//zy2018 set RA and Dec with the same value
```

Comment:
The documentation here: http://www.ioptron.com/v/ASCOM/RS-232_Command_Language2014_V2.0.pdf

specifies this format:

Command: “:RGnnn#”
Response: “1”
Sets the guiding rate n.nn * sidereal rate. nnn can only be entered in the range of [10, 90].

Reply:
We have a CEM25 and it is tested, the true command is  ":RGaabb#", guide rate of RA is 0.aa, guide rate of Dec is 0.bb

Reply:
Ok, there are two commands set. The older one is :RGAAA# and the new one is :RGAABB# I think it depends on the firmware.. can you post the firmware version for the CEM25? Is it 20171001 or earlier?

Reply:
I think no, the firmware version here:
HC:170518
RA:170106
Dec:170106
is it the reason that command is different?